starter-code/src/App.js

```diff
@@ -1,18 +1,123 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
+import 'bulma/css/bulma.css';
+import FoodBox from './components/FoodBox'
+import Search from './components/Search'
+import TotalsCheckout from './components/TotalsCheckout'
+import foods from './foods.json'
+import { v4 as uuidv4 } from 'uuid';
+
+const newFoods = foods.map(food => {
+
+food['id'] = uuidv4()
+
+return food
+})
 
 class App extends Component {
+
+state = {foods:newFoods,
+         selectedFoods:newFoods,
+         form: false,
+         search:'',
+         name: '',
+         calories: 0,
+         image: '',
+         quantity: 0,         
+        }
+
+
+handleSearchInput = (e) => {
+const selectedFoods = this.state.foods.filter((food) => {
+      return food.name.toLowerCase().includes(e.target.value);
+    });
+this.setState({
+      selectedFoods:selectedFoods,
```

Comment:
Just wanted to point out that you can also write the short-hand syntax `this.setState({search:e.target.value, selectedFoods})`